Lib/asyncio/selector_events.py

```diff
@@ -458,16 +458,15 @@ def _sock_sendall(self, fut, sock, data):
     async def sock_connect(self, sock, address):
         """Connect to a remote socket at address.
 
+        address must be a socket address tuple(i.e. (ipv4_address, port) for
```

Comment:
What about...

```suggestion
        Address must be a socket address tuple(i.e. (ipv4_address, port) for
```

?

Reply:
Hmm, I intended "address" to refer to the method argument. Maybe I should quote it between backticks, like \`address\`?

Reply:
ahhh yes! you must use `address` to avoid (like me) confusions